data/common.yaml

```diff
@@ -1,3 +1,3 @@
 ---
-cvmfs::repo_includepkgs: ~
```

Comment:
Better to just delete the line (the file) completely. 
Should allow the merge parameter case to work.